starfish/pipeline/segmentation/watershed.py

```diff
@@ -1,11 +1,13 @@
 import numpy as np
 import scipy.ndimage.measurements as spm
+from regional import many
```

Comment:
nit: prefer import regional, and then using `regional.many`, like in https://github.com/spacetx/starfish/pull/368/files#diff-54c1560b3315fe0c319c3bebcad83299

rationale: if it's not obvious what package something comes from (i.e., not a pretty standard library), I prefer a more qualified name.